src/templates/partials/fold-finsec.hbs

```diff
@@ -1,18 +1,14 @@
 <li id="finsec">
   <div>
-    <img class="background" src="/assets/img/landing/finsec/background.jpg">
```

Comment:
Again, why are we using an `<img>` where we can simply use CSS `background-color`??